react/react2/week1-exercise/app/blogs/page.jsx

```diff
@@ -0,0 +1,20 @@
+"use client";
+import { usePathname } from "next/navigation";
+
+function Blogs() {
+  const pathname = usePathname();
+  return (
+    <div>
+      <p className="mb-4 italic">Current pathname: {pathname}</p>
+      <p className="mb-4 text-xl">
+        Try changing the URL to something like
```

Comment:
This is a good one, try to see if you can create links so your users don't have to guess the route name 😀